MAUI/Radial-Menu/Overview.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: About MAUI Radial Menu control | Syncfusion
+description: Learn here all about introduction of Syncfusion MAUI Radial Menu (SfRadialMenu) control, its elements and more.
+platform: MAUI
+control: RadialMenu (SfRadialMenu)
+documentation: ug
+---
+
+# Overview of .NET MAUI RadialMenu (SfRadialMenu)
+
+The Syncfusion .NET MAUI SfRadialMenu displays a hierarchical menu in a circular layout, which is optimized for touch enabled devices. Typically, it is used as a context menu, and it can expose more menu items in the same space than traditional menus.
+
+## Key features
+
+`Drag` : The SfRadialMenu can be floated over the layout to avoid obscuring the content behind it.
+`Rotation` : SfRadialMenu supports rotating items.
+`FontIcon`: The built-in icon font option helps users add vector images that prevent the control from experiencing any image glitches often faced with traditional image icons.
+`Custom view` : SfRadialMenu supports the custom view like image.
+`Custom segments` : Complete customization options for the menu and its items, such as coloring, sizing,palcement and shapes, using the segmentation option.
```

Comment:
`Custom segments` : Complete customization options for the menu and its items, such as coloring, sizing, placement, and shapes, using the segmentation option.